src/olympia/devhub/tests/test_views_submit.py

```diff
@@ -110,11 +113,45 @@ def test_read_dev_agreement_skip(self):
         self.assert3xx(response, reverse('devhub.submit.distribution'))
 
 
-class TestCreateAddon(BaseUploadTest, TestCase):
+class TestAddonSubmitDistribution(TestCase):
     fixtures = ['base/users']
 
     def setUp(self):
-        super(TestCreateAddon, self).setUp()
+        super(TestAddonSubmitDistribution, self).setUp()
```

Comment:
This isn't a new class - it was just in the wrong order (you choose the distribution before you create the Addon).  See below in the file.